ClusterTest/run_stress_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export TEST_COMPILE_SCALA_VERSION=2.12
+export TEST_SPARK_CONNECTOR_VERSION=2.8.2
+export TEST_SPARK_VERSION=3.0
+export TEST_CLUSTERTEST_VERSION=1.0
+
+# Test Revision ID
+export TEST_REVISION_ID=1
```

Comment:
Can you add comment to explain TEST_REVISION_ID and when it should be bump up.

Reply:
Done!